### ISSUE TITLE
Should we go install nvm for node.js runtime?

### ISSUE THREAD

**Issue by srt32** [globality-corp/laptop#3]

**Comment by cdm-ium** [comment]
Yes. I think all the new people had to do this to get fermina to work for them. I.e. me + @sethisernhagen , I would assume @jligh probably needed to do it too.....

**Comment by adamhadani** [comment]
:+1:  - in particular with move to ES5/6 from coffeescript we require now node 5.x

**Comment by Phylodome** [comment]
:+1:
